docs/docs/cloud/deployment/deploy.md

```diff
@@ -0,0 +1,152 @@
+# LangGraph Deploy
+
+With LangGraph Deploy, you define a simple configuration file and it generates a Docker image with:
+
+🎁 Automatic HTTP prediction server: Your agent definition is used to dynamically generate a RESTful HTTP API.
+
+🥞 Automatic queue worker. Long-running agents or batch processing is best architected with a queue. LangGraph Deploy manages this out of the box.
+
+☁️ Persistence storage. Optimized Postgres Checkpointers and Stores are spun up and attached to your agent, giving it persistence. This enables memory and human-in-the-loop interation patterns.
+
+🚀 Ready for production. Deploy your model anywhere that Docker images run. Your own infrastructure, or LangGraph Cloud.
+
+## How it works
+
+### Define the Docker environment your agent runs in with `langgraph.json`:
+
+```
+{
+  "dependencies": ["./my_agent"],
+  "graphs": {
+    "agent": "./my_agent/agent.py:graph"
+  },
+  "env": ".env"
```

Comment:
Confused about how `env` works actually given that we're making a baked image, presumably configuration is provided later on when running container?